buildSrc/src/main/java/Config.kt

```diff
@@ -111,6 +115,7 @@ object Config {
         val mockitoInline = "org.mockito:mockito-inline:3.10.0"
         val awaitility = "org.awaitility:awaitility-kotlin:4.1.0"
         val mockWebserver = "com.squareup.okhttp3:mockwebserver:4.9.0"
+        val mockWebserver3 = "com.squareup.okhttp3:mockwebserver:3.14.9"
```

Comment:
Curious why do we need two versions? Android vs Java?

Reply:
apollo probably depends on an older version